src/js/themes/base.js

```diff
@@ -1036,6 +1040,67 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // colors: [] || colors: ['graph-0', 'graph-1', 'graph-2', 'graph-3'],
       // extend: undefined,
     },
+    notification: {
+      container: {
+        pad: { horizontal: 'small', vertical: 'xsmall' },
+        background: {
+          color: 'background-front',
+        },
+      },
+      toast: {
+        container: {
+          elevation: 'medium',
+          round: 'xsmall',
+          width: 'medium',
+        },
+      },
+      iconContainer: {
+        pad: { right: 'small' },
+      },
+      textContainer: {
+        gap: 'medium',
+      },
+      title: {
+        weight: 'bold',
```

Comment:
Similar comment to "any Box props" but for Text. Reference: https://github.com/grommet/grommet/blob/89e8c5fea22324804c806d5871d5237a8cd99573/src/js/themes/base.js#L1226

```suggestion
      title: {
       // any Text props
        weight: 'bold',
```